modules/cart/src/Form/AddToCartForm.php

```diff
@@ -143,8 +155,7 @@ public function buildEntity(array $form, FormStateInterface $form_state) {
     $entity = parent::buildEntity($form, $form_state);
     // Now that the purchased entity is set, populate the title and price.
     $entity->setTitle($entity->getPurchasedEntity()->getLineItemTitle());
-    // @todo Remove once the price calculation is in place.
-    $entity->unit_price = $entity->getPurchasedEntity()->price;
+    $entity->setUnitPrice($this->chainPriceResolver->resolve($entity->getPurchasedEntity()));
```

Comment:
Maybe we should just remove this, and skip injecting the resolver? This is supposed to be done by the order refresh process, which needs to land today as well.